### PR TITLE
Add new resource /teachers/{username}/events

### PR DESCRIPTION
**Pull request by jirutka**

### CODE REVIEW

File: app/api/events_endpoints.rb [hunk 1]
```diff
@@ -119,6 +119,24 @@ def represent(dataset)
       end
     end
 
+    desc 'Filter events by teacher'
+    segment :teachers do
+      # XXX: username validation is in PeopleEndpoints
```
Comment (jnv) [inline app/api/events_endpoints.rb]: Is it? People endpoint is people endpoint. Different segment.

**Comment by flexik** [comment]
Except for the username validation it looks good. :+1:

**Comment by jirutka** [comment]
@jnv, updated, review plz

**Comment by jnv** [comment]
:shipit:

**Pull request merged**